example/src/BUILD.bazel

```diff
@@ -66,6 +66,12 @@ py_library(
     srcs = ["unused_import.py"],
 )
 
+filegroup(
+    name = "unused_import_filegroup",
+    srcs = ["unused_import.py"],
```

Comment:
since your stated motivation is to allow `.pyi` files, could you change the example to illustrate that? I don't want this file to be confusing to users. Also having an input in the `srcs` of two neighboring rules isn't a pattern we want users to copy.